app/views/distributions/show.html.erb

```diff
@@ -5,7 +5,7 @@
         <% content_for :title, "#{@distribution.storage_location.name} - Distributions - #{current_organization.name}" %>
         <h1>
           Distribution
-          <small>for <%= @distribution.storage_location.name %></small>
+          <small>from <%= @distribution.storage_location.name %> to <%=@distribution.partner.name %></small>
```

Comment:
Could we add some visual distinction on the partner name? Maybe bold it so we can see it is different from the storage location?

Reply:
Hi @edwinthinks ! Make sense :) done :)